versioned_docs/version-1.0.0/install-kubeslice/yaml/yaml-register-worker-clusters.mdx

```diff
@@ -200,7 +195,7 @@ file to install the Slice Operator. By completing these steps, the worker cluste
 integrate with the KubeSlice Controller and utilize its features.
 
 :::info
-** THIS STEP NEEDS TO BE PERFORMED ON ALL WORKER CLUSTERS IN THE CONFIGURATION. **
+This step needs to be performed on all worker clusters in the configuration.
```

Comment:
Let this be in upper case for more visibility.

Reply:
done